test/Microsoft.AspNet.Mvc.Core.Test/ModelBinding/SimpleTypeModelBinderTest.cs

```diff
@@ -189,6 +189,31 @@ public static TheoryData<Type> ConvertableTypeData
             Assert.True(bindingContext.ModelState.ContainsKey("theModelName"));
         }
 
+        [Theory]
+        [InlineData("0", "0")]
+        [InlineData("1", "Value1")]
+        [InlineData("13", "Value1, Value4, Value8")]
+        [InlineData("Value1", "Value1")]
+        [InlineData("Value8, Value4", "Value4, Value8")]
+        public async Task BindModel_BindsFlagsEnum(string flagsEnumValue, string expected)
+        {
+            // Arrange
+            var bindingContext = GetBindingContext(typeof(FlagsEnum));
+            bindingContext.ValueProvider = new SimpleValueProvider
+            {
+                { "theModelName", flagsEnumValue }
+            };
+
+            var binder = new SimpleTypeModelBinder();
+
+            // Act
+            var result = await binder.BindModelAsync(bindingContext);
+
+            // Assert
+            Assert.True(result.IsModelSet);
+            Assert.Equal(expected, result.Model.ToString());
```

Comment:
Do we need to tostring here? can't we just compare to the enum value? (Pass in expected as an int and cast it)